src/RA_Dlg_MemBookmark.cpp

```diff
@@ -833,17 +832,17 @@ std::wstring Dlg_MemBookmark::ImportDialog()
 
     CComPtr<IFileOpenDialog> pDlg;
 
-    HRESULT hr;
+    auto hr{ S_OK };
```

Comment:
another abstracted HRESULT

Reply:
Most of it will be gone later but I'll unabstract it for clarity when I can. I understand not everyone will know what s_ok is.